### PR TITLE
Updates airports.json File

### PR DESCRIPTION
**Pull request by ashok9180**

### CODE REVIEW

**Review by tah — CHANGES_REQUESTED** [review]
A few points:
- Why are almost all lines in `airports.json` removed?
- Let's not commit the `package-lock.json` in a lib like this

**Comment by fullstackdeveloper918** [comment]
> Why are almost all lines in airports.json removed?

Because the old file is merged with new data .....through code.

> Let's not commit the package-lock.json in a lib like this

Sure

File: package.json [hunk 1]
```diff
@@ -24,6 +24,8 @@
   "dependencies": {
```
Comment (tah) [inline package.json]: These should all be `devDependencies` right? They are not needed when using and importing the lib.

**Review by tah — COMMENTED** [review]
Could we combine/merge the `newConvert.js` script with `convert.js`?
1. Read and parse `.dat` file
2. Read and parse `.csv` file
3. Combine and reduce to unique entries
4. Write json file

File: package.json (hunk 1 above)
Comment (ashok9180) [reply package.json]: Yeah you are right.

**Review by tah — APPROVED** [review]

**Pull request merged**